21.7 fb bcs weight report

Ohsudev opened the pull request:
#### Rationale


#### Related Pull Requests


#### Changes
Provide column option to generate weight of monkey at time of most recent BCS Score.

Ohsudev committed:
Created a new program that will provide weight values based on another table's record date

Ohsudev committed:
Allow users to display the animal weight column to any NHP grid reports.

Ohsudev committed:
Merge branch 'release21.7-SNAPSHOT' into 21.7_fb_BCSWeightReport

Ohsudev committed:
Provide column option to generate weight of monkey at time of most recent BCS Score.

Ohsudev commented:
Hi Josh. Can you please add G. Jones as reviewer after he resets his login to github.

labkey-jeckels commented:
@Ohsudev as you already discovered, you could add his normal GitHub user without any problems. Once he's logged into it he'll be able to review and approve as usual

jonesgaohsu reviewed (approved):
Looks good Lets move forward

Ohsudev committed:
Provide column option to generate weight of monkey at time of most recent BCS Score.

Ohsudev committed:
Merge branch 'release21.7-SNAPSHOT' into 21.7_fb_BCSWeightReport

Ohsudev committed:
Provide an optional weight's data when running
a grid report that contains BCS scores.  Provide weight's data that corresponds to the nearest weight date to the most recent BCS Score date.

Ohsudev committed:
Provide an optional weight's data when running
a grid report that contains BCS scores.  Provide weight's data that corresponds to the nearest weight date to the most recent BCS Score date.

Ohsudev committed:
Modified to include an option to display the weight of a monkey closest to the most recent BCS Score date.

Ohsudev committed:
Merge branch 'release21.7-SNAPSHOT' into 21.7_fb_BCSWeightReport

Ohsudev committed:
Modified to include an option to display the weight of a monkey closest to the most recent BCS Score date.

Ohsudev commented:
Hi Josh.  Can you please finalized your review of this PR so I can move it forward into default build.

Thank you.
Raymond

Ohsudev commented:
Thank you.

From: Josh Eckels ***@***.***>
Sent: Monday, February 14, 2022 4:36 PM
To: LabKey/onprcEHRModules ***@***.***>
Cc: Raymond Blasa ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [LabKey/onprcEHRModules] 21.7 fb bcs weight report (PR #344)


@labkey-jeckels approved this pull request.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/LabKey/onprcEHRModules/pull/344*pullrequestreview-882341074__;Iw!!Mi0JBg!fSxGQJwOwPh46u43TLfq3BEZOiFYU4goi-NeTboZj2jKmnQb13CacH-oHJKV5w$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ASHU2YHI6F6MKUBYBPABNBDU3GNYZANCNFSM5LLICKLQ__;!!Mi0JBg!fSxGQJwOwPh46u43TLfq3BEZOiFYU4goi-NeTboZj2jKmnQb13CacH8uZNcBbw$>.
Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.com/v3/__https:/apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675__;!!Mi0JBg!fSxGQJwOwPh46u43TLfq3BEZOiFYU4goi-NeTboZj2jKmnQb13CacH8tlo_QjQ$> or Android<https://urldefense.com/v3/__https:/play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign*3Dnotification-email*26utm_medium*3Demail*26utm_source*3Dgithub__;JSUlJSU!!Mi0JBg!fSxGQJwOwPh46u43TLfq3BEZOiFYU4goi-NeTboZj2jKmnQb13CacH9Cv5P3dg$>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
